### DISCUSSION TITLE
How to stop carousel for last item when align="start"

### DISCUSSION THREAD

**Discussion by branislav-brincko** [mantinedev/mantine#6459] [Q&A]
When align option is set to start, the last item can be "scrolled" to the left of the carousel container: 

![image](https://github.com/mantinedev/mantine/assets/118365024/31a3b14a-b9d6-44de-af8a-0eedb872e285)

I want it to stop like this: 

![image](https://github.com/mantinedev/mantine/assets/118365024/a6a606e8-cc1d-4fe2-96ef-f4e3706d8fe2)

Is there a way to achieve this? Basically i want to achieve exactly this example https://www.embla-carousel.com/examples/predefined/#slides-per-view. 

My current code:

![image](https://github.com/mantinedev/mantine/assets/118365024/539b02a2-5787-4b0f-90ab-1f8f69cff8ad)

**Comment by rtivital** [comment]
Try `containScroll="trimSnaps"`
